modules/sql/managed-instance/README.md

```diff
@@ -592,3 +592,112 @@ module managedInstance './sql/managed-instance/main.bicep' = {
 
 </details>
 <p>
+
+<h3>Example 3: Varbac</h3>
```

Comment:
I'd recommend to call the folder just something like 'vulnAssm' or similar. That we run RBAC in the test is true, but I wonder if this should be the one thing we highlight :D

Reply:
I like it, updated to 'vulnAssm'.